pystiche_papers/utils/misc.py

```diff
@@ -238,3 +239,13 @@ def select_url_from_csv(
             f"{fieldname}: {value}" for fieldname, value in zip(fieldnames, config)
         )
         raise RuntimeError(msg) from error
+
+
+def license(license: str, original: Optional[str] = None) -> str:
```

Comment:
So this thing does nothing if `original` is not passed? What is the point to use it in that case?